tools/cool_to_hic/cool_to_hic.xml

```diff
@@ -0,0 +1,75 @@
+<tool name="cool_to_hic" id="cool_to_hic" version="0.0.8" profile="22.05">
+  <!--Source in git at: https://github.com/fubar2/galaxy_tf_overlay-->
+  <!--Created by toolfactory@galaxy.org at 01/02/2024 07:02:54 using the Galaxy Tool Factory.-->
+  <description>Convert .cool to .hic binary for JBrowse viewing or other use</description>
```

Comment:
Maybe remove jbrowse from the description? It's a normal generic converter.

Reply:
done